gui/util/range_slider.py

```diff
@@ -0,0 +1,294 @@
+"""
+Range slider, extended QWidget slider for napari-gui.
+"""
+
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+
+class QRangeSlider(QtWidgets.QWidget):
+    """
+    QRangeSlider class, super class for QVRangeSlider and QHRangeSlider.
+    """
+
+    rangeChanged = QtCore.pyqtSignal(float, float)
+
+    def __init__(self, slider_range, values, parent=None):
+        QtWidgets.QWidget.__init__(self, parent)
+        self.bar_width = 4
```

Comment:
I kinda squirm when I see "magic values" like this one. Can we use instead an init value or a config or a global "constant"?